Framework/Algorithms/src/InterpolatingRebin.cpp

```diff
@@ -200,27 +201,18 @@ void InterpolatingRebin::outputYandEValues(
 *  @throw invalid_argument if any output x-values are outside the range of input
 *x-values
 **/
-void InterpolatingRebin::cubicInterpolation(const HistogramData::BinEdges &xOld,
-                                            const MantidVec &yOld,
-                                            const MantidVec &eOld,
-                                            const HistogramData::BinEdges &xNew,
-                                            MantidVec &yNew,
-                                            MantidVec &eNew) const {
-  // Make sure y and e vectors are of correct sizes
+void InterpolatingRebin::cubicInterpolation(const Histogram &oldHistogram,
```

Comment:
If it does not do anything, it implies that the workspace is already in the expected state, i.e., the histograms have `XMode::Counts` (instead of `XMode::Frequencies`). See if you used `setCounts()` somewhere where you should not have.